taxi/models.py

```diff
@@ -31,3 +32,11 @@ class Car(models.Model):
 
     def __str__(self):
         return self.model
+
+
+class VisitCounter(models.Model):
+    num_visits = models.PositiveIntegerField(default=0)
+
+    def increment(self):
+        self.num_visits += 1
+        self.save()
```

Comment:
Why do you need it here? Your counter should be on `session` layer, so you shouldn't create this model